helm/gort/templates/gort/_helpers.tpl

```diff
@@ -0,0 +1,34 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "gort.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "gort.fullname" -}}
```

Comment:
This results in resources all being prefixed "chart-gort". Is that intentional?

Note that this would need a corresponding change in the Tiltfile to add a port forward to the resource.

Reply:
Yes, intentional. 🙂

This is Helm chart best practice for any application so that multiple installations (via helm install FOO) don't conflict with one another. This is especially important for Gort, though, because it has to dynamically look up its own Endpoint resources to issue API calls to itself.

Reply:
I'll make that Tilt change!